content/release-notes/1.23.0.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2020-11-13T13:30:14-08:00
+linktitle: "1.23.0"
+title: "1.23.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<features>}}
+* Added new template function [KotsVersion](https://kots.io/reference/template-functions/static-context/#kotsversion)
+* Added [collectors](https://troubleshoot.sh/docs/collect/ceph/) and [analyzers](https://troubleshoot.sh/docs/analyze/ceph-status/) for Ceph.
+* Added a [collector](https://troubleshoot.sh/docs/collect/collectd/) for Collectd RRD files.
+{{</features>}}
+
+{{<changes>}}
+* Added a button for redeploying failed deployments.
```

Comment:
@jgruica Is it only for failed deployments? I thought you could redeploy successful deployments too.

Reply:
@markpundsack this is just for failed deployments

Reply:
Ah, ok. I'd still like to be able to redeploy things we "think" were successful, but weren't somehow. But I'll make a separate CH for that.